librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -1447,6 +1447,28 @@ commands:
           - name: skip
             type: RZ_CMD_ARG_TYPE_RZNUM
             optional: true
+      - name: p=r
+        summary: Print rising and falling entropy.
+        cname: print_rising_and_falling_entropy
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
```

Comment:
please use `state` instead of the raw modes, it will help with the function body as well. I recommend splitting it into smaller static functions

Reply:
What does it mean to "use `state` instead of the raw modes."

Reply:
Do I need to add any `subcommands` for this? And on what basis will I make subcommands?

Reply:
Then what do I need to change?

Reply:
Add `type: RZ_CMD_DESC_TYPE_ARGV_STATE`

Reply:
I have already added that. 
![image](https://github.com/rizinorg/rizin/assets/76243531/8b613974-2d6b-43c9-ae71-83f617e8e665)